content/docs/programs/immunizations.md

```diff
@@ -94,13 +71,35 @@ To add a new Vaccine Course, click the `Add vaccine course` button in the top ri
 
 ![vaccine course: add button](/docs/programs/images/vaccine_course_add_button.png)
 
-This will open a new window, where you can enter the name of the new Vaccine Course.
+This will open a new window, where you can enter the details of the new Vaccine Course, including its name, details like its wastage rate, and the dose schedule.
 
 ![vaccine course: create modal](/docs/programs/images/vaccine_course_add.png)
 
+#### Doses
+
+To add doses to the Vaccine Course, click the `(+) Dose` button. This will add a new row to the dose schedule, where you can enter the details of the dose.
+
+![vaccine course: dose edit table](/docs/programs/images/vaccine_course_dose_edit.png)
+
+The Dose Schedule table is divided into 6 columns:
+
+| Column                  | Description                                                                                                              |
+| :---------------------- | :----------------------------------------------------------------------------------------------------------------------- |
+| **Dose #**              | The number of the dose in the schedule                                                                                   |
+| **Label**               | The label of the dose, to appear in the [Vaccination Card](/docs/programs/program-module#vaccination-cards)              |
+| **From age**            | Minimum recommended age to receive the dose (in years and months)                                                        |
+| **To age**              | Maximum recommended age to receive the dose (in years and months)                                                        |
+| **Custom age label**    | Custom age label to be displayed in the Vaccination Card. If not defined, the default is the `From age` (e.g. `6 years`) |
+| **Min interval (days)** | Minimum number of days before the patient should receive the next dose                                                   |
+
 - Click `OK` once you are happy with your changes
-  - You will be redirected to the detail view of your newly created Vaccine Course
-- OR: Click `Cancel` to close the window if you've changed your mind
+- OR: Click `Cancel` to close the window and discard your changes
```

Comment:
Before these two sentences something like....

Once you are happy with the vaccination details, you can: 
- Click
- Click

and delete the OR